pkg/model/awsmodel/bastion.go

```diff
@@ -270,11 +271,11 @@ func (b *BastionModelBuilder) Build(c *fi.ModelBuilderContext) error {
 		// Here we implement the bastion CNAME logic
 		// By default bastions will create a CNAME that follows the `bastion-$clustername` formula
 		t := &awstasks.DNSName{
-			Name:      s(bastionPublicName),
+			Name:      fi.String(bastionPublicName),
```

Comment:
I think I agree with this change (it's more verbose but clearer _not_ to use the helper functions), but it might have been easier to do this as a follow-on PR or separate commit, just from a change readability perspective.

Reply:
Yeah. This change was mainly done because `awstasks` does not have these helper functions, and I didn't see the point of copying them.